### DISCUSSION TITLE
Grupo

### DISCUSSION THREAD

**Discussion by lguerra98** [rramosp/ai4eng.v1#157] [UDEA Students]
Buenas noches, yo pretendia hacer el trabajo solo pero encontre unos compañeros los cuales estan todavia buscando grupo por unos inconvenientes que tuvieron, todavia puedo crear un grupo con ellos? gracias

**Comment by rramosp** [comment]
sí, adelante ... pero registrad grupo y repositorios como todo el mundo

On Fri, Jul 1, 2022 at 11:59 PM lguerra98 ***@***.***> wrote:

> Buenas noches, yo pretendia hacer el trabajo solo pero encontre unos
> compañeros los cuales estan todavia buscando grupo por unos inconvenientes
> que tuvieron, todavia puedo crear un grupo con ellos? gracias
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/157>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIMFMFGDZEPQJPEMJYDVR7EBLANCNFSM52OPBD7A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
